### DISCUSSION TITLE
Advise on content writing style for playbook articles

### DISCUSSION THREAD

**Discussion by yantantether** [nhsbsa/nhsbsa-digital-playbook#272] [General Playbook Ideas]
Its going to be difficult to have a consistent writing style across the playbook when we have so many contributors from different professions with different perspectives.

Can Content Design provide an article spelling out how article content should be written? We can probably link to Gov guidance as well, but I think it'd be helpful to have some specific guidance.

**Comment by nhsbsa-sthil** [comment]
Ha! Very timely post @yantantether! We were just talking about this in our Senior UCD chat yesterday 😄 

I wonder whether we should perhaps also make it part of the process / checklist of whether an item is 'ready' to be taken from a 'discussion' into an 'issue'? i.e. similar to our projects where we have a 'definition of ready'
